### PR TITLE
Fixes and suggestions for section 5

### PR DESCRIPTION
**Pull request by eguiraud**
In particular I removed references to "computation graph" because without introducing
the concept (possibly with a picture) it will be completely lost on most
readers, and changed "declarative" to "lazy".

### CODE REVIEW

**Comment by stwunsch** [comment]
I thought about adding one of the images from doxygen or our other talks, e.g., the ATLAS courses. But at the end I didn't know where to put it ...

File: _episodes/05-rdataframe.md [hunk 1]
```diff
@@ -33,15 +33,18 @@ ROOT tries to make parallelization as simple as possible for you. For this reaso
 ```python
 import ROOT
 
-# Enable multi-threading
-ROOT.EnableImplicitMT(0)
```
Comment (stwunsch) [inline _episodes/05-rdataframe.md:37]: That was on purpose so they can later rerun on 2 or more threads and see that they run faster!

Comment (stwunsch) [reply _episodes/05-rdataframe.md:37]: At the end, all code snipplet should be copy pastable one after another and result in a working program

Comment (stwunsch) [reply _episodes/05-rdataframe.md:37]: You like that (implicit) idea here or shall we do something else?

File: _episodes/05-rdataframe.md [hunk 2]
```diff
@@ -75,26 +78,29 @@ df_mass = df_os.Define("Dimuon_mass", "ComputeInvariantMass(Muon_pt, Muon_eta, M
...
 
 ```python
 # Produce plot
 ROOT.gStyle.SetOptStat(0)
 ROOT.gStyle.SetTextFont(42)
 c = ROOT.TCanvas("c", "", 800, 700)
-h.GetXaxis().SetTitleSize(0.04); h.GetYaxis().SetTitleSize(0.04)
+# The contents of one of the dataframe results are accessed for the first time here:
+# this is when all results will actually be produced!
+h.GetXaxis().SetTitleSize(0.04);
```
Comment (stwunsch) [inline _episodes/05-rdataframe.md]: Either remove the semicolon or move the lines again together!

**Review by stwunsch — COMMENTED** [review]
What about the `ROOT.EnableImplicitMT(0)` thingy?

File: _episodes/05-rdataframe.md (hunk 1 above)
Comment (eguiraud) [reply _episodes/05-rdataframe.md:37]: I don't think the idea worked, `EnableImplicitMT(0)` is equivalent to `EnableImplicitMT()` :D

but to keep things copy-pastable the second `EnableImplicitMT` command can be commented out

File: _episodes/05-rdataframe.md (hunk 2 above)
Comment (eguiraud) [reply _episodes/05-rdataframe.md]: semicolon removed!

File: _episodes/05-rdataframe.md (hunk 1 above)
Comment (stwunsch) [reply _episodes/05-rdataframe.md:37]: Ah damn, I thought the `0` disables it! So well, now it's a win win situation ;) I'll make a issue and fix it later!

**Comment by stwunsch** [comment]
Thanks, great!

**Pull request merged**